src/app/app.scss

```diff
@@ -48,10 +48,27 @@ body {
     user-select: text;
 }
 
+.divider {
+    border-bottom: 1px solid lightgray;
+    width: inherit;
+}
+
 .no-padding {
     padding: 0px !important;
 }
 
+.no-overflow{
+    width: 100%;
+    max-width: 20ch;
```

Comment:
What is the ch unit? Can we use px or % here for consistency?

Reply:
The ch is characters (technically measured as the width of the "0" character). I used it because this set the maximum character length of names and teams inside lists but there is no reason we cant use % so I changed it.